apps/web-v2/src/lib/auth/supabase-client.ts

```diff
@@ -27,15 +27,18 @@ export function getSupabaseClient() {
   supabaseInstance = createBrowserClient<Database>(supabaseUrl, supabaseKey, {
     cookies: {
       get(name) {
+        if (typeof document === "undefined") return undefined;
```

Comment:
nit, `window` is the standard check. `document` can be accessed via `window` (`window.document`) so you typically just check for `window`
```suggestion
        if (typeof window === "undefined") return undefined;
```